### ISSUE TITLE
Issue while deploying EBS CSI driver using [ aws-ebs-csi-driver:latest ] image on Graviton2 Powered Worker Nodes

### ISSUE THREAD

**Issue by utk231** [kubernetes-sigs/aws-ebs-csi-driver#901]
/kind bug

What happened? Following the instructions at https://github.com/kubernetes-sigs/aws-ebs-csi-driver to deploy the driver on Amazon EKS cluster running with Graviton2 based worker nodes. The driver pods ebs-csi-controller, ebs-csi-node keep crashing.

What you expected to happen? - CSI-Controller & CSI-Node pod to be in `Running` state but goes in `CrashLoopBackOff` immediately after start

How to reproduce it (as minimally and precisely as possible)?

```
1. git clone https://github.com/kubernetes-sigs/aws-ebs-csi-driver.git
2. Use aws-ebs-csi-driver:latest image in aws-ebs-csi-driver/deploy/kubernetes/base/controller.yaml & aws-ebs-csi-driver/deploy/kubernetes/base/node.yaml files
3. kubectl apply -k aws-ebs-csi-driver/deploy/kubernetes/base/
```

Anything else we need to know?: This works fine on Graviton2 instances when image `aws-ebs-csi-driver:v1.0.0` is used in both files.

Environment:

- Kubernetes version (use `kubectl version`): Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.16-eks-7737de", GitCommit:"7737de131e58a68dda49cdd0ad821b4cb3665ae8", GitTreeState:"clean", BuildDate:"2021-03-10T21:33:25Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}

- Driver version: Driver image aws-ebs-csi-driver:latest used in https://github.com/kubernetes-sigs/aws-ebs-csi-driver/tree/master/deploy/kubernetes/base

**Comment by touseefspeaks** [comment]
**While deploying ebs driver on eks cluster(1.18) i am facing with following issue:**
**Kubectl version**:  `Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.14", GitCommit:"89182bdd065fbcaffefec691908a739d161efc03", GitTreeState:"clean", BuildDate:"2020-12-18T12:02:35Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}`

` kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=master" --kubeconfig kubeconfig_cloud-policy      
error: AccumulateTarget: rawResources failed to read Resources: Load from path poddisruptionbudget-controller.yaml failed: evalsymlink failure on 'C:\Users\XXXX\AppData\Local\Temp\kustomize-474850325\deploy\kubernetes\base\poddisruptionbudget-controller.yaml' : CreateFile C:\Users\XXXX\AppData\Local\Temp\kustomize-474850325\deploy\kubernetes\base\poddisruptionbudget-controller.yaml: The system cannot find the file specified.`

**Comment by utk231** [comment]
@touseefspeaks Please remove following entries from `kustomization.yaml` file under `aws-ebs-csi-driver/deploy/kubernetes/base/` as it's not present on repository at the location https://github.com/kubernetes-sigs/aws-ebs-csi-driver/tree/master/deploy/kubernetes/base

- poddisruptionbudget-controller.yaml
- poddisruptionbudget-snapshot-controller.yaml

**Comment by krmichel** [comment]
The files should have been there.  They got dropped in a rebase.  There is now a PR to add them.

**Comment by utk231** [comment]
@krmichel The original issue is that when `aws-ebs-csi-driver:latest` is specified as image source in controller and node YAML Files, Driver pods keep going to `CrashLoopBackOff` state after starting. Will that issue be fixed in this PR #906 ?

**Comment by krmichel** [comment]
No, sorry.  I had too many emails this morning and thought that these were two different issues.  The PR will only address the missing PDBs.  I have unlinked it from this issue.
